test/functions/bradc/paramThis/funParamThis2.good

```diff
@@ -1 +1 @@
-funParamThis2.chpl:1: error: 'this' intents can only be applied to methods
+funParamThis2.chpl:1: error: Cannot apply a 'this' intent to a standalone function
```

Comment:
Personally, I like the old error message better and wouldn't be opposed to changing the error message and thisIntentStandalone.good. Do you think it's worth checking with Mike, Vass, or Brad to see if they have a preference?